index.idyll

```diff
@@ -483,10 +483,18 @@ Digital streaming is not the only instance where environmentally harmful aspects
 Tech is rarely perceived as environmentally toxic, but here's a surprising fact: [span className:"emphasized"]Santa Clara County, the heart of "Silicon Valley", has the most EPA classified "superfund" (highly polluted) sites in the nation[/span].
 These [link text:"23 locations" url:"https://www.theatlantic.com/technology/archive/2019/09/silicon-valley-full-superfund-sites/598531/#:~:text=Santa%20Clara%20County%20has%2023,these%20chemicals%20may%20be%20impossible." target:"_blank"/] may be impossible to fully clean. Silicon Valley is primarily to blame.
 
+[var name:"isTouchScreen" value:false /]
+[TouchscreenDetector isTouchScreen:isTouchScreen /]
+
+[derived name:"superfundCaption" value:`"The 23 superfund sites of Santa Clara County. " +
+  (isTouchScreen ? 'Trigger the "Enable Touch" button, then tap each site for more information.' :
```

Comment:
Any better word besides "Trigger"? Maybe "Select" or "Press" or simply use "Tap" twice in that sentence?

Reply:
👍 "press". Thank you!